### PR TITLE
Add missing descriptions to PrimitiveMesh and SoftBody3D

### PR DESCRIPTION
**Pull request by Mickeon**
This PR ...💤💤...
... This brings them to 100% completion.

Pressure coefficient source: https://arxiv.org/ftp/physics/papers/0407/0407003.pdf

May require the insight from the Physics team for the missing SoftBody3D descriptions.

### CODE REVIEW

File: doc/classes/SoftBody3D.xml [hunk 1]
```diff
@@ -100,11 +101,13 @@
 			[b]Note:[/b] Object A can detect a contact with object B only if object B is in any of the layers that object A scans. See [url=$DOCS_URL/tutorials/physics/physics_introduction.html#collision-layers-and-masks]Collision layers and masks[/url] in the documentation for more information.
 		</member>
 		<member name="damping_coefficient" type="float" setter="set_damping_coefficient" getter="get_damping_coefficient" default="0.01">
+			The body's damping coefficient. Higher values will slow down the body more noticeably when forces are applied.
 		</member>
 		<member name="disable_mode" type="int" setter="set_disable_mode" getter="get_disable_mode" enum="SoftBody3D.DisableMode" default="0">
 			Defines the behavior in physics when [member Node.process_mode] is set to [constant Node.PROCESS_MODE_DISABLED]. See [enum DisableMode] for more details about the different modes.
 		</member>
 		<member name="drag_coefficient" type="float" setter="set_drag_coefficient" getter="get_drag_coefficient" default="0.0">
+			The body's drag coefficient. Higher values make the body bouncier.
```
Comment (AThousandShips) [inline doc/classes/SoftBody3D.xml:110]: Is this correct? Drag is normally about air resistance

In fact it seems it's not actually used anywhere at least not in the standard physics

Comment (Mickeon) [reply doc/classes/SoftBody3D.xml:110]: It... seems to be used for SoftBody3D, but could this be placebo? As physics are, the code is not exactly clear.

No, you're right, this was a slip on my part. This has got to be about air resistance.

Comment (AThousandShips) [reply doc/classes/SoftBody3D.xml:110]: Nowhere in the code is this used as of right now, seems to have been used in Bullet but not implemented yet here

Comment (Mickeon) [reply doc/classes/SoftBody3D.xml:110]: Oh boy.

**Comment by hemu0069** [comment]
nice one

**Review by clayjohn — APPROVED** [review]
The PrimitiveMesh description looks good!

The SoftBody stuff also seems fine, but I'm less knowledgeable about that

**Review by rburing — APPROVED** [review]

**Comment by YuriSizov** [comment]
Cherry-picked for 4.2.2.

**Comment by YuriSizov** [comment]
Cherry-picked for 4.1.4.

**Pull request merged**